WordPress/openverse-catalog
# Remove tsv_to_postgres_loader_overwrite

## Fixes


Fixes #273 by @AetherUnbound

## Description


This PR consolidates the `tsv_to_postgres_loader` DAG in order to remove the `overwrite`-specific DAG. As part of this effort I also removed some constants and added the docstring as DAG documentation.

![image](https://user-images.githubusercontent.com/10214785/140553180-dbce4d76-9348-46a3-9b20-361cc5ae60a4.png)

## Testing Instructions

1. `just test`
1. `just run airflow dags list` - observe that `tsv_to_postgres_loader` is present and `tsv_to_postgres_loader_overwrite` is not

## Checklist


[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
